content/v1.12/concepts/providers.md

```diff
@@ -172,31 +172,49 @@ The Crossplane community has more
 {{< /hint >}}
 ### Provider conditions
 
+Crossplane uses a standard set of `Conditions` for Providers.  
 View the conditions of a provider under their `Status` with 
-`kubectl describe provider`.
+`kubectl describe provider`. 
 
-Providers have the following possible conditions:
+```yaml
+kubectl describe provider
```

Comment:
this is a truncated output right? is it important to say that some of the output is ommitted for brevity, so as to not confuse readers why their output may be much longer than this sample output here?

i've always wondered about that...